tests/unit/components/sl-date-range-picker-test.js

```diff
@@ -0,0 +1,352 @@
+        'Start date input placeholder is empty when startDatePlaceholder is undefined'
+    );
+
+    assert.strictEqual(
+        component.get( 'endDatePlaceholder' ),
+        undefined,
+        'endDatePlaceholder is undefined by default'
+    );
+
+    assert.strictEqual(
+        this.$( '.sl-daterange-end-date input' ).prop( 'placeholder' ),
+        '',
+        'End date input placeholder is empty when endDatePlaceholder is undefined'
+    );
+});
+
+test( 'startDatePlaceholder is accepted as a parameter', function( assert ) {
+    let placeholderText = 'lorem ipsum';
+    let component = this.subject({ startDatePlaceholder: placeholderText });
+
+    assert.equal(
+        component.get( 'startDatePlaceholder' ),
+        placeholderText,
+        'startDatePlaceholder was accepted as a parameter'
+    );
+
+    assert.equal(
+        this.$( '.sl-daterange-start-date input' ).prop( 'placeholder' ),
+        placeholderText,
+        'Start date input placeholder was passed through correctly'
+    );
+
+    assert.strictEqual(
+        component.get( 'label' ),
+        undefined,
+        'label is undefined by default'
+    );
+
+    assert.strictEqual(
+        this.$( 'label' )[0],
+        undefined,
+        'No label is created when label is undefined'
+    );
+
+    assert.strictEqual(
+        this.$( '.sl-daterange-end-date input' ).prop( 'placeholder' ),
+        '',
+        'End date input placeholder is empty when endDatePlaceholder is undefined'
+    );
+
+    assert.strictEqual(
+        component.get( 'endDatePlaceholder' ),
+        undefined,
+        'endDatePlaceholder is undefined by default'
+    );
+});
+
+test( 'endDatePlaceholder is accepted as a parameter', function( assert ) {
+    let placeholderText = 'lorem ipsum';
+    let component = this.subject({ endDatePlaceholder: placeholderText });
+
+    assert.equal(
+        component.get( 'endDatePlaceholder' ),
+        placeholderText,
+        'endDatePlaceholder was accepted as a parameter'
+    );
+
+    assert.equal(
+        this.$( '.sl-daterange-end-date input' ).prop( 'placeholder' ),
+        placeholderText,
+        'End date input placeholder was passed through correctly'
+    );
+
+    assert.strictEqual(
+        component.get( 'label' ),
+        undefined,
+        'label is undefined by default'
+    );
+
+    assert.strictEqual(
+        this.$( 'label' )[0],
+        undefined,
+        'No label is created when label is undefined'
+    );
+
+    assert.strictEqual(
+        component.get( 'startDatePlaceholder' ),
+        undefined,
+        'startDatePlaceholder is undefined by default'
+    );
+
+    assert.strictEqual(
+        this.$( '.sl-daterange-start-date input' ).prop( 'placeholder' ),
+        '',
+        'Start date input placeholder is empty when startDatePlaceholder is undefined'
+    );
+});
+
+test( 'label, startDatePlaceholder, and endDatePlaceholder can be accepted as parameters together', function( assert ) {

```

Comment:
We have not traditionally tested these type of scenarios unless there are specific combinations of parameters that invoke specific functionality.  I recommend removing this test.